tests/unit/assessment/test_dashboard.py

```diff
@@ -30,15 +30,15 @@ def test_dashboard(mocker):
     ws.query_visualizations.create.return_value = Visualization(id="abc")
     ws.dashboard_widgets.create.return_value = Widget(id="abc")
     installer = WorkspaceInstaller(ws)
-    local_query_files = installer._find_project_root() / "src/databricks/labs/ucx/assessment/queries"
+    local_query_files = installer._find_project_root() / "src/databricks/labs/ucx/queries"
```

Comment:
Add a test for a secondary dashboard

Reply:
This already reads the SQL files contained in the project, which means it creates multiple dashboards.